tests/js/spec/components/modals/widgetViewerModal.spec.tsx

```diff
@@ -763,5 +769,87 @@ describe('Modals -> WidgetViewerModal', function () {
           ' minmax(90px, auto) minmax(90px, auto) minmax(575px, auto)',
       });
     });
+
+    it('uses provided issuesData and does not make an issues requests', async function () {
+      await renderModal({initialData, widget: mockWidget, issuesData: []});
+      expect(issuesMock).not.toHaveBeenCalled();
+    });
+
+    it('makes issues requests when table is sorted', async function () {
+      await renderModal({
+        initialData,
+        widget: mockWidget,
+        issuesData: [],
+      });
+      expect(issuesMock).not.toHaveBeenCalled();
+      userEvent.click(screen.getByText('events'));
+      await waitFor(() => {
+        expect(issuesMock).toHaveBeenCalled();
+      });
+    });
+  });
+
+  describe('Table Widget', function () {
+    let eventsv2Mock;
+    const mockQuery = {
+      conditions: 'title:/organizations/:orgId/performance/summary/',
+      fields: ['title', 'count()'],
+      aggregates: ['count()'],
+      columns: ['title'],
+      id: '1',
+      name: 'Query Name',
+      orderby: '',
+    };
+    const mockWidget = {
+      title: 'Test Widget',
+      displayType: DisplayType.TABLE,
+      interval: '5m',
+      queries: [mockQuery],
+      WidgetType: WidgetType.DISCOVER,
+    };
+    beforeEach(function () {
+      eventsv2Mock = MockApiClient.addMockResponse({
```

Comment:
yesterday I got this [feedback](https://github.com/getsentry/sentry/pull/33302#discussion_r845160077) in my PR and was wondering if we should do the same here